roles/edpm_ovn_bgp_agent/templates/rootwrap.conf.j2

```diff
@@ -0,0 +1,27 @@
+# Configuration for ovn-bgp-agent-rootwrap
```

Comment:
Do we have to create this file ? If the file is installed inside the container image by rpm then we don't have to create this file, assuming the default rootwrap.conf should work.

The file is named j2 but this is not really a j2 template and nothing can be confugired.

Reply:
it was done this way so that it can be configured if needed. Can we keep it or should we remove it and only add it once there is a need for configuration?

Reply:
I'd advice we defer implementation unless we find real requirements.

If we keep these files now then we have to maintain these files (eg. if the default files in repo/rpms are updated then we have to update these files in edpm-ansible accordingly). I'd prefer avoiding additional maintenance effort without real gain.